### ISSUE TITLE
QRiS 0.1.6 fails on install - `No module named 'sklearn' `

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/QRiS#137]
On my Windows 11 Machine with QGIS 3.24.3 and Python 3.9.5, install from zip fails with this message:

```
Couldn't load plugin 'qris_deploy' due to an error when calling its classFactory() method 

ModuleNotFoundError: No module named 'sklearn' 
Traceback (most recent call last):
  File "C:\PROGRA~1/QGIS32~1.3/apps/qgis/./python\qgis\utils.py", line 423, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "C:\Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qris_deploy\__init__.py", line 40, in classFactory
    from .src.qris_toolbar import QRiSToolbar
  File "C:\PROGRA~1/QGIS32~1.3/apps/qgis/./python\qgis\utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:\Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qris_deploy\src\qris_toolbar.py", line 40, in 
    from .view.frm_dockwidget import QRiSDockWidget
  File "C:\PROGRA~1/QGIS32~1.3/apps/qgis/./python\qgis\utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:\Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qris_deploy\src\view\frm_dockwidget.py", line 62, in 
    from .frm_stream_gage_docwidget import FrmStreamGageDocWidget
  File "C:\PROGRA~1/QGIS32~1.3/apps/qgis/./python\qgis\utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:\Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qris_deploy\src\view\frm_stream_gage_docwidget.py", line 10, in 
    from sklearn.metrics import dcg_score
  File "C:\PROGRA~1/QGIS32~1.3/apps/qgis/./python\qgis\utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
ModuleNotFoundError: No module named 'sklearn'


Python version: 3.9.5 (tags/v3.9.5:0a7dcbd, May  3 2021, 17:27:52) [MSC v.1928 64 bit (AMD64)] 
QGIS version: 3.24.3-Tisler Tisler, cf22b74e 

Python Path:
C:/PROGRA~1/QGIS32~1.3/apps/qgis/./python
C:/Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins
C:/PROGRA~1/QGIS32~1.3/apps/qgis/./python/plugins
C:\Program Files\QGIS 3.24.3\bin\python39.zip
C:\PROGRA~1\QGIS32~1.3\apps\Python39\DLLs
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib
C:\Program Files\QGIS 3.24.3\bin
C:\PROGRA~1\QGIS32~1.3\apps\Python39
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\GDAL-3.4.3-py3.9-win-amd64.egg
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\win32
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\win32\lib
C:\PROGRA~1\QGIS32~1.3\apps\Python39\lib\site-packages\Pythonwin
C:/Users/joemw/AppData/Roaming/QGIS/QGIS3\profiles\default/python
```

**Comment by philipbaileynar** [comment]
Should be fixed in 0.1.7alpha.

**Comment by joewheaton** [comment]
Thanks for quick turn around.
